src/pkg/selectors/sharepoint_test.go

```diff
@@ -394,6 +411,8 @@ func (suite *SharePointSelectorSuite) TestSharePointScope_MatchesInfo() {
 				SharePoint: &details.SharePointInfo{
 					ItemType: details.SharePointItem,
 					WebURL:   test.infoURL,
+					Created:  now,
+					Modified: now,
```

Comment:
Can you use two separate times here, so that the test would fail if the Created before/after was used for the Modified time, and vice versa?  I think that's why some of the other bugs slipped through for you.

Reply:
Resolved [eafba68](https://github.com/alcionai/corso/pull/2679/commits/eafba68ee2997fb557a01707a7506e31b7fdd219)